src/app/components/core/notifications/notifications.component.css

```diff
@@ -29,8 +29,5 @@ button {
 }
 
 #refresh-check {
```

Comment:
as we have moved this to src/styles.css do we  still need it?

Reply:
Yes, because `margin: auto;` only requires in Notification page, rest properties of `#refresh-check` are inherited from style.css.